src/main/java/org/broadinstitute/dsm/db/dao/kit/BSPKitDao.java

```diff
@@ -0,0 +1,82 @@
+package org.broadinstitute.dsm.db.dao.kit;
```

Comment:
I wonder if we should rename that to KitDao and lose the BSP.

Reply:
If it is just used for the dummy kit creation maybe we should add Dummy to the name